inngest/src/main/kotlin/com/inngest/InngestFunctionConfigBuilder.kt

```diff
@@ -12,6 +12,7 @@ class InngestFunctionConfigBuilder {
     private var name: String? = null
     private var triggers: MutableList<InngestFunctionTrigger> = mutableListOf()
     private var concurrency: MutableList<Concurrency>? = null
+    private var retries: Int? = null
```

Comment:
for the default number of retries, should we instead make this non-optional and set it to 3? Then we wouldn't need the elvis operator in `buildSteps`